Build fails on Windows 10

Seems to be an issue with the webpack, any guidance?

$ npm run build

> squoosh@1.0.2 build C:\Squoosh\squoosh
> webpack -p

   76 modules

ERROR in ./src/codecs/processor-worker.ts (./node_modules/worker-plugin/dist/loader.js?{"name":"processor-worker"}!./src/codecs/processor-worker.ts)
Module build failed (from ./node_modules/worker-plugin/dist/loader.js):
ModuleNotFoundError: Module not found: Error: Can't resolve 'fs' in 'C:\Squoosh\squoosh\codecs\imagequant'
    at factory.create (C:\Squoosh\squoosh\node_modules\webpack\lib\Compilation.js:797:10)
    at factory (C:\Squoosh\squoosh\node_modules\webpack\lib\NormalModuleFactory.js:397:22)
    at resolver (C:\Squoosh\squoosh\node_modules\webpack\lib\NormalModuleFactory.js:130:21)
    at asyncLib.parallel (C:\Squoosh\squoosh\node_modules\webpack\lib\NormalModuleFactory.js:224:22)
    at C:\Squoosh\squoosh\node_modules\neo-async\async.js:2817:7
    at C:\Squoosh\squoosh\node_modules\neo-async\async.js:6783:13
    at normalResolver.resolve (C:\Squoosh\squoosh\node_modules\webpack\lib\NormalModuleFactory.js:214:25)
    at doResolve (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:184:12)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:15:1)
    at resolver.doResolve (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\UnsafeCachePlugin.js:37:5)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:15:1)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:12:1)
    at resolver.doResolve (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\DescriptionFilePlugin.js:42:38)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn41 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:381:1)
    at resolver.doResolve (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\ModuleKindPlugin.js:23:37)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:15:1)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:15:1)
    at args (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\forEachBail.js:30:14)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:15:1)
    at resolver.doResolve (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\UnsafeCachePlugin.js:37:5)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:15:1)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:12:1)
    at resolver.doResolve (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\DescriptionFilePlugin.js:42:38)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn42 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:393:1)
    at hook.callAsync (C:\Squoosh\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)
    at _fn0 (eval at create (C:\Squoosh\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:12:1)

same here

`λ npx webpack serve                                                                                          
i ｢wds｣: Project is running at http://localhost:8080/les                                                     
i ｢wds｣: webpack output is served from /                                                                     
i ｢wds｣: Content not from webpack is served from E:\laragon\www\squoosh\src                                  
i ｢wds｣: 404s will fallback to /index.html                                                                   
× ｢wdm｣:    146 modules                                                                                      
                                                                                                             
ERROR in ./src/codecs/processor-worker.ts (./node_modules/worker-plugin/dist/loader.js?{"name":"processor-wor
ker"}!./src/codecs/processor-worker.ts)                                                                      
Module build failed (from ./node_modules/worker-plugin/dist/loader.js):                                      
ModuleNotFoundError: Module not found: Error: Can't resolve 'fs' in 'E:\laragon\www\squoosh\codecs\imagequant
'                                                                                                            
    at factory.create (E:\laragon\www\squoosh\node_modules\webpack\lib\Compilation.js:797:10)                
    at factory (E:\laragon\www\squoosh\node_modules\webpack\lib\NormalModuleFactory.js:397:22)               
    at resolver (E:\laragon\www\squoosh\node_modules\webpack\lib\NormalModuleFactory.js:130:21)              
    at asyncLib.parallel (E:\laragon\www\squoosh\node_modules\webpack\lib\NormalModuleFactory.js:224:22)     
    at E:\laragon\www\squoosh\node_modules\neo-async\async.js:2817:7                                         
    at E:\laragon\www\squoosh\node_modules\neo-async\async.js:6783:13                                        
    at normalResolver.resolve (E:\laragon\www\squoosh\node_modules\webpack\lib\NormalModuleFactory.js:214:25)
    at doResolve (E:\laragon\www\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:184:12)               
    at hook.callAsync (E:\laragon\www\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)           
    at _fn0 (eval at create (E:\laragon\www\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anon
ymous>:15:1)                                                                                                 
    at resolver.doResolve (E:\laragon\www\squoosh\node_modules\enhanced-resolve\lib\UnsafeCachePlugin.js:37:5
)                                                                                                            
    at hook.callAsync (E:\laragon\www\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)           
    at _fn0 (eval at create (E:\laragon\www\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anon
ymous>:15:1)                                                                                                 
    at hook.callAsync (E:\laragon\www\squoosh\node_modules\enhanced-resolve\lib\Resolver.js:238:5)           
    at _fn0 (eval at create (E:\laragon\www\squoosh\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anon
ymous>:12:1)                                                                                                 
    at resolver.doResolve (E:\laragon\www\squoosh\node_modules\enhanced-resolve\lib\DescriptionFilePlugin.js:
42:38)                                                                                                       
i ｢wdm｣: Failed to compile.                                                                                  `

Fedora 29
```sh
➜  squoosh git:(master) ✗ npm -v
6.4.1
➜  squoosh git:(master) ✗ node -v
v10.12.0
```

```sh
➜  squoosh git:(master) ✗ npm run build

> squoosh@1.0.2 build /home/neewbee/demos/squoosh
> webpack -p

   76 modules

ERROR in /home/neewbee/demos/squoosh/src/codecs/generic/quality-option.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/codecs/generic/quality-option.tsx(3,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/codecs/imagequant/options.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/codecs/imagequant/options.tsx(5,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/codecs/mozjpeg/options.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/codecs/mozjpeg/options.tsx(5,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/codecs/optipng/options.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/codecs/optipng/options.tsx(6,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/codecs/resize/options.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/codecs/resize/options.tsx(6,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/codecs/webp/options.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/codecs/webp/options.tsx(5,24)
      TS2307: Cannot find module '../../components/Options/style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/App/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/App/index.tsx(4,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/Options/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/Options/index.tsx(3,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/Output/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/Output/index.tsx(5,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/Output/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/Output/index.tsx(9,29)
      TS2307: Cannot find module './custom-els/TwoUp/styles.css'.

ERROR in /home/neewbee/demos/squoosh/src/components/Output/custom-els/TwoUp/index.ts
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/Output/custom-els/TwoUp/index.ts(2,25)
      TS2307: Cannot find module './styles.css'.

ERROR in /home/neewbee/demos/squoosh/src/components/checkbox/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/checkbox/index.tsx(2,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/compress/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/compress/index.tsx(5,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/compress/custom-els/MultiPanel/index.ts
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/compress/custom-els/MultiPanel/index.ts(1,24)
      TS2307: Cannot find module './styles.css'.

ERROR in /home/neewbee/demos/squoosh/src/components/custom-els/LoadingSpinner/index.ts
./src/components/custom-els/LoadingSpinner/index.ts
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/custom-els/LoadingSpinner/index.ts(1,25)
      TS2307: Cannot find module './styles.css'.

ERROR in /home/neewbee/demos/squoosh/src/components/expander/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/expander/index.tsx(2,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/intro/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/intro/index.tsx(14,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/range/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/range/index.tsx(2,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/results/FileSize.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/results/FileSize.tsx(3,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/results/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/results/index.tsx(3,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/components/select/index.tsx
[tsl] ERROR in /home/neewbee/demos/squoosh/src/components/select/index.tsx(2,24)
      TS2307: Cannot find module './style.scss'.

ERROR in /home/neewbee/demos/squoosh/src/custom-els/RangeInput/index.ts
[tsl] ERROR in /home/neewbee/demos/squoosh/src/custom-els/RangeInput/index.ts(3,24)
      TS2307: Cannot find module './styles.css'.

ERROR in /home/neewbee/demos/squoosh/src/lib/SnackBar/index.ts
[tsl] ERROR in /home/neewbee/demos/squoosh/src/lib/SnackBar/index.ts(1,24)
      TS2307: Cannot find module './styles.css'.
Error parsing bundle asset "/home/neewbee/demos/squoosh/build/first-interaction.0aabb.js": no such file
Error parsing bundle asset "/home/neewbee/demos/squoosh/build/main-app.38ebc.js": no such file
Error parsing bundle asset "/home/neewbee/demos/squoosh/build/offliner.31821.js": no such file
Error parsing bundle asset "/home/neewbee/demos/squoosh/build/wc-polyfill.b195e.js": no such file

No bundles were parsed. Analyzer will show only original module sizes from stats file.

Webpack Bundle Analyzer saved report to /home/neewbee/demos/squoosh/build/report.html
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! squoosh@1.0.2 build: `webpack -p`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the squoosh@1.0.2 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/neewbee/.npm/_logs/2018-11-15T01_29_35_024Z-debug.log
```

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -169,7 +169,11 @@ module.exports = function (_, env) {
       ]
     },
     plugins: [
-      new webpack.IgnorePlugin(/(fs|crypto|path)/, /\/codecs\//),
+      new webpack.IgnorePlugin(
+        /(fs|crypto|path)/,
+        new RegExp(`${path.sep}codecs${path.sep}`)
+      ),
+
       // Pretty progressbar showing build progress:
       new ProgressBarPlugin({
         format: '\u001b[90m\u001b[44mBuild\u001b[49m\u001b[39m [:bar] \u001b[32m\u001b[1m:percent\u001b[22m\u001b[39m (:elapseds) \u001b[2m:msg\u001b[22m\r',

